STYLE.md

```diff
@@ -75,6 +75,7 @@ following abbreviations we assume readers will already know.
 
 | Use | Don't use | Notes |
 |-|-|-|
+| anti fee sniping | anti-fee sniping, anti-fee-sniping | |
```

Comment:
fwiw git grepping "anti-fee" and "fee-snip" returns quite a few of both

Reply:
Yeah, which is why I decided to choose a particular version.  Really, all the alternatives here seem awful to me, so feel free to let me know if you prefer a different version (or, better yet, can think of a clever new term that could credibly gain adoption if we use it repeatedly).

Reply:
Hm, as you've surely done, I looked online and in the codebase and see "fee sniping" and "anti-fee-sniping". I actually like the term and it seems to date back to at least 2015. I come to the same conclusion that you do, e.g. hyphenless, or alternatively use the two that we see.